analysis/study_definition_rates.py

```diff
@@ -80,6 +79,13 @@
             "incidence": 0.4
         }
     ),
+    incidence=patients.satisfying(
+        """
+        incid
+        AND incid_date = prevalence_date
+        """
+    ),## will this let me establish incidence? 
```

Comment:
This looks correct to me. Alternatively you could se dynamic dates to check for [previous occurrences](https://github.com/opensafely/antidepressant-prescribing-lda/blob/e50bd7479bf87c10947b0a1aec9b30b2f7518924/analysis/study_definition.py#L111-L122)

Reply:
Amazing, thank you, I have used this way, better than mine